src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -5,11 +5,13 @@ import java.util.UUID
 
 import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
 import com.typesafe.config.ConfigFactory
+import io.grpc.Status.Code
```

Comment:
unused? let's try to keep google things out of leo model

Reply:
Unused import, thanks